buildscripts/lvm-driver/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 FROM alpine:3.12
 RUN apk add --no-cache lvm2 lvm2-extra util-linux device-mapper
-RUN apk add --no-cache btrfs-progs xfsprogs e2fsprogs e2fsprogs-extra
+RUN apk add --no-cache btrfs-progs xfsprogs xfsprogs-extra e2fsprogs e2fsprogs-extra
```

Comment:
question: Is this package required to resize xfs filesystem?

Reply:
yes.